### PR TITLE
[BACKPORT] #19725, #19801

### PR DESCRIPTION
**Pull request by access2rohit**
Backporting #19725, #19801 as a port of effort #19911

### CODE REVIEW

**Comment by access2rohit** [comment]
@josephevans please review

File: tools/pip/doc/CU112_ADDITIONAL.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+Prerequisites
+-------------
+This package supports Linux and Windows platforms. You may also want to check:
+- [mxnet-cu110](https://pypi.python.org/pypi/mxnet-cu110/) with CUDA-11.0 support.
```
Comment (josephevans) [inline tools/pip/doc/CU112_ADDITIONAL.md:21]: Should there be a 11.2 version as well?

File: ci/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,215 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (josephevans) [inline ci/docker/docker-compose.yml:1]: We shouldn't need to bring this file into v1.x, since we aren't using docker-compose.

**Review by josephevans — APPROVED** [review]
LGTM, thanks.

**Pull request merged**